model/config.go

```diff
@@ -2557,7 +2601,7 @@ func (fs *FileSettings) isValid() *AppError {
 		return NewAppError("Config.IsValid", "model.config.is_valid.file_driver.app_error", nil, "", http.StatusBadRequest)
 	}
 
-	if len(*fs.PublicLinkSalt) < 32 {
+	if fs.PublicLinkSalt != nil && *fs.PublicLinkSalt != "" && len(*fs.PublicLinkSalt) < 32 {
```

Comment:
Same comment as above.

Reply:
I relaxed the condition slightly, but we still need to check for empty string.